README.md

```diff
@@ -6,4 +6,5 @@ After hearing the sound signal (clap) from the victim, the robot should send an
 
 The project was created using ESP and Arduino UNO microcontrollers and also necessary sensors. Everything was powered by 4 AA batteries and a power bank.
 
+Click here for more information: https://www.bitehack.best.krakow.pl/
```

Comment:
is this the main page of the competition?

Reply:
yep